app/core/components/LabeledTextField/index.tsx

```diff
@@ -1,21 +1,21 @@
-import type { PropsWithoutRef } from "react";
```

Comment:
Seems like your prettier configuration is not using the default we have defined. Since it is removing the semicolons. Hmmmmm, now that I remember, could you remove lines 23 and 22 to 25 of [package.json](https://github.com/wizeline/remix-project-lab/blob/main/package.json#L22-L25)? I think they are the ones causing all the trouble.